Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+pipeline {
+    agent {node {label 'builder'}}
+
+    stages {
+        stage('checking out the code') {
+            steps {
+                git 'https://github.com/droritzz/hello-flask.git'
```

Comment:
you don't need to specify the URL if the code comes from the same repo as Jenkinsfile. Just use `checkout scm` - https://www.jenkins.io/doc/pipeline/steps/workflow-scm-step/

Reply:
done